cmd/limactl/edit.go

```diff
@@ -40,12 +43,16 @@ func editAction(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	if inst.Status == store.StatusRunning {
+		return errors.New("Cannot edit a running instance")
+	}
```

Comment:
We should take a lock, but can be another PR

https://github.com/lima-vm/lima/blob/v0.8.1/pkg/lockutil/lockutil.go